test/locales/en/test_context_plural.json

```diff
@@ -1,5 +1,5 @@
 {
   "first": "first",
-  "first_context1_plural": "first context1 plural",
+  "first_context1_other": "first context1 plural",
   "first_context2_plural_2": "first context2 plural 2"
```

Comment:
Not sure what the context2 is, but `plural_2` looks fishy with the new plural suffixes

Reply:
`context2` is just from the test. You are right that `plural_2` is weird, that said it doesn't even show in the test. @Elindorath  Could you change it to `first_context2_two` and update the test in parser.js:642 to see if the plural is properly picked up?

Reply:
It starts to smell like a bug. That's also why I was curious if changing it to the new plural form changed the outcome of the test. @Elindorath would you mind testing?